### DISCUSSION TITLE
Wrong team names for 1st round of every match

### DISCUSSION THREAD

**Discussion by emteka** [akiver/cs-demo-analyzer#1] [General]
![image](https://github.com/akiver/cs-demo-manager/assets/45483666/79fce8a7-f8fc-4c3a-8ae7-ae7357034399)

all the rounds != 1 have correct team names

PS. please, create discord for this app, I'd love to share with community how I visualize data from team perspective

**Comment by akiver** [comment]
I transferred the discussion here and moved it as an issue because it's a demo analysis issue.
Please use the GitHub issue template next time and fill in all fields - it helps debug the problem.

Your demos are coming from eBot?

**Comment by akiver** [comment]
It's fixed
